bulbs/contributions/views.py

```diff
@@ -151,6 +152,38 @@ def get_queryset(self):
         return qs
 
 
+class LineItemReportingViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+
+    renderer_classes = (
+        LineItemRenderer,
+    ) + tuple(
```

Comment:
I like this `tuple` use instead of just parens with an easy-to-forget trailing comma